api/test/sections/sections-crud.spec.ts

```diff
@@ -3,30 +3,18 @@ import { TestManager } from 'api/test/utils/test-manager';
 
 describe('Page Sections API', () => {
   let testManager: TestManager<unknown>;
-  let authToken: string;
 
   beforeAll(async () => {
     testManager = await TestManager.createTestManager();
     await testManager.clearDatabase();
-    const { jwtToken } = await testManager.setUpTestUser();
-    authToken = jwtToken;
   });
 
   afterAll(async () => {
     await testManager.clearDatabase();
     await testManager.close();
   });
 
-  it('Should allow anonymous users to read page sections', async () => {
-    // When
-    const res = await testManager.request().get('/sections');
-
-    // Then
-    expect(res.status).toBe(200);
-    expect(res.body.data).toBeDefined();
-  });
-
-  it('Should allow authenticated users to read page sections with its widgets', async () => {
```

Comment:
the endpoint is public and we implicitly retrieve base widgets with each section, meaning there is no behaviour change depending on wether the user is authenticated or not